utils/docker/run-coverity.sh

```diff
@@ -39,6 +39,18 @@ set -e
 # Prepare build environment
 ./prepare-for-build.sh
 
+# Coverity doesn't support gcc 7+, because of newly introduced _Float* types.
+# It manifests as number of "compilation units ready for analysis"
+# below 85% threshold and compile errors like these in cov-int/build-log.txt:
+#"/usr/include/math.h", line 381: error #20: identifier "_Float32" is undefined
+#  # define _Mdouble_		_Float32
+# Work around this by replacing gcc symlink with gcc-6. Setting CC/CXX
+# environment variables is unfortunately not enough.
+sudo rm /usr/bin/gcc
```

Comment:
the proper way to do this is to use `update-alternatives`.

Reply:
When I wrote this I checked update-alternatives and couldn't find gcc alternative, so I assumed it's not handled.
But I checked it now once again and found out there's cc and c++ alternatives for /usr/bin/gcc and /usr/bin/g++...
Will fix.

Reply:
Unfortunately:
- only gcc and clang are supported
- it actually changes cc symlink, not gcc and Coverity seems to use gcc